Cargo.toml

```diff
@@ -81,6 +82,7 @@ evm-debug = ["ethcore/evm-debug"]
 evm-debug-tests = ["ethcore/evm-debug-tests"]
 slow-blocks = ["ethcore/slow-blocks"]
 final = ["ethcore-util/final"]
+sstore = ["ethcore-secstore"]
```

Comment:
I think it would be best to enable it by default (or build it on the CI), otherwise it's hard to detect any possible breaking changes causing compilation errors in that crate, since people don't alter the features and builds are not including this as well.